tools/check_release.py

```diff
@@ -432,17 +483,26 @@ def get_latest_library_versions(repo_path):
                         # Internal compiler error thus retry
                         continue
                     else:
-                        # Genuine compilation error, thus print it out
-                        logging.error("\t\tError: %s\n", mesg)
+                        # Actual error thus move on to next compilation
+                        failures.append([test, target])
+                        break
                                     
                 passes += (int)(result)
                 break
             else:
-                logging.error("\t\tProgram/Target compilation failed due to internal errors. Removing from considered list!\n")
+                logging.error("\t\tCompilation failed due to internal errors.\n")
+                logging.error("\t\tSkipping test/target combination!\n")
```

Comment:
Can we drop the `\t\t`? If you want speciallized output styling there is always [Config options](https://docs.python.org/3/library/logging.html?highlight=logging#logging.basicConfig).